databricks/koalas/groupby.py

```diff
@@ -2021,7 +2021,10 @@ def nunique(self, dropna=True):
                 + F.when(F.count(F.when(col.isNull(), 1).otherwise(None)) >= 1, 1).otherwise(0)
             )
 
-        should_include_groupkeys = isinstance(self, DataFrameGroupBy)
+        # The behavior of GroupBy.nunique has been changed since pandas 1.1.0
```

Comment:
FYI: https://pandas.pydata.org/pandas-docs/dev/whatsnew/v1.1.0.html#consistency-across-groupby-reductions

Reply:
I prefer the new behavior. 👍

Reply:
Me, too. It seems like more reasonable ! :)